apps/nextjs-website/src/lib/types/tutorialData.ts

```diff
@@ -1,6 +1,7 @@
 import { Path } from '@/lib/types/path';
 
 export type Tutorial = {
+  readonly coomingsoon?: boolean;
```

Comment:
```suggestion
  readonly comingSoon: boolean;
```
what do you think to call it `comingSoon` and to remove the optional? optional to a boolean could be handle as "if present -> true; if not -> false"

Reply:
I rename the attribute as suggested :+1:  But i think is better mark attribute as optional bacause is only temporary flag as coomingSoon content and i think it will be removed in feature.